sdk/eventgrid/azure-eventgrid/setup.py

```diff
@@ -64,16 +64,14 @@
     classifiers=[
```

Comment:
lines 25-36 can be removed: https://github.com/Azure/azure-sdk-for-python/blob/main/doc/dev/packaging.md#note-on-checking-old-azure-packages

Reply:
oh interesting, I did not know. Maybe we can ask Laurent about this and send him a link to one of those questions?